docs/src/content/docs/fr/guides/components.mdx

```diff
@@ -129,6 +129,50 @@ Ajoutez l'attribut `stagger` pour décaler verticalement la deuxième colonne de
 
 :::
 
+### Link Cards
+
+Utilisez le composant `<LinkCard>` pour créer un lien vers différentes pages.
+
+Une `<LinkCard>` nécessite un `title` et un attribut [`href`](https://developer.mozilla.org/fr/docs/Web/HTML/Element/a#href). Vous pouvez optionellement inclure une courte `description` ou d'autres attributs de lien tels que `target`.
+
+Regroupez plusieurs composants `<LinkCard>` dans `<CardGrid>` pour afficher les cartes côte à côte lorsqu'il y a suffisamment d'espace.
```

Comment:
Noticing this is the only place we refer to these as “cartes”. Should we also update the headings to use that too? Currently they use the English “Cards” and “Link Cards”.

I looked up the [French government design system](https://www.systeme-de-design.gouv.fr/elements-d-interface/composants/carte) and they also use “Carte” although on their index page they always include English and French terms side by side:
<img width="829" alt="Three links from the French government design system components index showing titles including both French and English names for each component." src="https://github.com/withastro/starlight/assets/357379/47c8337d-bff3-4696-9d61-d9a4257f4090">

So maybe our headings should do something similar like “Cartes (Cards)” and “Cartes liens (Link Cards)”?